tests/Resources/config/routes.yml

```diff
@@ -0,0 +1,3 @@
+_controllers:
+  resource: ../Controller/
```

Comment:
Oooohh, so this is how you define attribute controllers, I was searching for something similar recently for the legacy controller Good to know!! Maybe I'll change it because the attribute configuration allows to specify a `priority` argument, and it seems to be the only type of configuration that allows this I wanted to make sure the generic wildcard legacy controller is always last in the router matching

I'll try this in a dedicated PR